gradle/plugins/common/src/main/kotlin/junitbuild.java-library-conventions.gradle.kts

```diff
@@ -17,13 +17,46 @@ plugins {
 	id("org.openrewrite.rewrite")
 }
 
+/**
+ * activeRecipe("org.openrewrite.java.RemoveUnusedImports") // there are (non); fails to analyze
```

Comment:
the test is working but the run still makes the change and fails the dryRun afterwards.

- https://github.com/openrewrite/rewrite/pull/5738